src/controllers/api.controller.ts

```diff
@@ -25,6 +25,18 @@ export const claimPost = async (
         ...rawClaim
       }
     });
+
+    // update cliam subject with this format https:/live.linkedtrust.us/claims/${claimId}
+    const claimId = claim.id;
+    const claim_subject = `https:/live.linkedtrust.us/claims/${claimId}`;
```

Comment:
hey remember @0marSalah we talked about having an encapsulated function

makeClaimUrl(claim_id)

that will be used EVERYWHERE in the code where we construct a subject.  this will prevent inconsistencies.  you must search for all places where this happens.  the above url is fine but of course fix https://